### PR TITLE
 EPMRPP-38599 Sauce Labs responsive layout control

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import ReactObserver from 'react-event-observer';
 import Fullscreen from 'react-full-screen';
+import { ReflexContainer, ReflexSplitter, ReflexElement } from 'react-reflex';
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx]: `import { ReflexContainer, ReflexSplitter, ReflexElement } from 'react-reflex/dist/commonjs';`

File: app/package.json [hunk 2]
```diff
@@ -70,6 +70,7 @@
     "react-lazyload": "2.5.0",
     "react-popper": "1.3.3",
     "react-redux": "5.0.7",
+    "react-reflex": "^3.0.16",
```
Comment (DiscoElevator) [inline app/package.json]: `"react-reflex": "3.0.16",`

File: app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx [hunk 3]
```diff
@@ -73,13 +75,20 @@ export class SauceLabsSection extends Component {
...
     const { isFullscreenMode } = this.state;
     const Content = (
-      <div className={cx('section-content-wrapper')}>
-        <VideoSection
-          observer={this.observer}
-          isFullscreenMode={isFullscreenMode}
-          onToggleFullscreen={this.toggleFullscreenMode}
-        />
-        <JobInfoSection observer={this.observer} isFullscreenMode={isFullscreenMode} />
+      <div>
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx]: Looks like this div can be removed

File: app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx]: Fixed

File: app/package.json (hunk 2 above)
Comment (AmsterGet) [reply app/package.json]: Fixed

File: app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx (hunk 3 above)
Comment (AmsterGet) [reply app/src/pages/inside/logsPage/sauceLabsSection/sauceLabsSection.jsx]: Fixed

**Pull request merged**